users/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db   import models
+
+from core.models import TimeStampModel
+
+class User(TimeStampModel):
+    name      = models.CharField(max_length=50)
+    email     = models.EmailField(max_length=200, unique=True)
+    address   = models.TextField()
+    user_name = models.CharField(max_length=50, unique=True)
```

Comment:
보통 유저네임은 `_` 없이 `username` 의 형태로 사용합니다.
```suggestion
    username = models.CharField(max_length=50, unique=True)
```